docs/connected_orgs.rst

```diff
@@ -0,0 +1,155 @@
+
+
+
+Use a Custom Connected App
+----------------------------
+CumulusCI uses a preconfigured Connected App to authenticate to Salesforce orgs that use OAuth2.
+In most cases this preconfigured app is all you need to authenticate into orgs.
+To control the Connected App for specific security or compliance requirements (such as adding a private key to sign a certificate connected with the configuration, or enforcing restrictions on user activity), create your own Connected App and configure CumulusCI to use it when connecting to orgs.
+
+To create a custom Connected App, run the ``connected_app`` task, and then manually `edit its configuration <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_connected_app.htm>`_ to suit your requirements.
+
+.. important::
+
+    Make sure to create the Connected App in a production org!
+
+
+This command will create a Connected App in the DevHub org connected to ``sfdx`` with the label ``cumulusci`` and set it as the ``connected_app`` service in CumulusCI.
+
+.. code-block:: console
+
+    $ cci task run connected_app --label cumulusci --connect true
+
+After the Connected App has been created, verify that it's connected to CumulusCI.
+
+.. code-block:: console
+
+    $ cci service list
+    +Services--------------------------------------------------------------------------------------------------------+
+    | Name           Description                                                                          Configured |
+    +----------------------------------------------------------------------------------------------------------------+
+    | connected_app  A Connected App is required to connect to and run commands against persistent orgs.  ✔          |
+    | devhub         Configure which SFDX org to use as a Dev Hub for creating scratch orgs               ✔          |
+    | github         Configure connection for github tasks, e.g. Create Release                           ✔          |
+    | metaci         Connect with a MetaCI site to run builds of projects from this repository                       |
+    | metadeploy     Connect with a MetaDeploy site to publish installers from this repository            ✔          |
+    | apextestsdb    Configure connection for ApexTestsDB tasks, e.g. ApextestsdbUpload                              |
+    | saucelabs      Configure connection for saucelabs tasks.                                                       |
+    +----------------------------------------------------------------------------------------------------------------+
+
+To edit the Connected App's OAuth scopes:
+
+#. In Lightning Experience, go to Setup --> Apps --> Apps Manager.
+#. Click the arrow on the far right side of the row that pertains to the newly created Connected App.
+#. Click "Edit."
+#. Add or remove OAuth scopes as desired.
+
+For a full list of options, see the :ref:`connected_app` task reference documentation.
```

Comment:
These task links don't seem to be rendering as expected.

Reply:
ya, odd. The syntax looks correct to me though... 🤔